cmd/api/src/daemons/datapipe/datapipe.go

```diff
@@ -133,8 +133,8 @@ func resetCache(cacher cache.Cache, cacheEnabled bool) {
 	}
 }
 
-func (s *Daemon) ingestAvailableTasks() {
-	if ingestTasks, err := s.db.GetAllIngestTasks(); err != nil {
+func (s *Daemon) ingestAvailableTasks(ctx context.Context) {
+	if ingestTasks, err := s.db.GetAllIngestTasks(ctx); err != nil {
```

Comment:
I'm probably just confused, but can you please provide details on why there's two different contexts in this function? This line uses the newly added parameter, but calls down below on 153, 158, and 162 are using the context on the receiver Daemon. Do we even need to pass in the parameter if the context is already available on the receiver?

Reply:
No we do not and I completely missed that for these Daemon functions. Good catch, updating!